frontend/src/containers/ReportStep/DoraMetricsChart/PipelineSelector/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { PipelinesSelectContainer } from '@src/containers/ReportStep/DoraMetricsChart/PipelineSelector/style';
+import { Autocomplete, Box, ListItemText, TextField, Tooltip } from '@mui/material';
+import { getEmojiUrls, removeExtraEmojiName } from '@src/constants/emojis/emoji';
+import { EmojiWrap, StyledAvatar } from '@src/constants/emojis/style';
+import { Z_INDEX } from '@src/constants/commons';
+import React, { useState } from 'react';
+
+interface Props {
+  options: string[];
+  value: string;
+  onUpDatePipeline: (value: string) => void;
+  title: string;
+}
+
+export default function PipelineSelector({ options, value, onUpDatePipeline, title }: Props) {
+  const label = '';
```

Comment:
there is a `EMPTY` string in the constant.ts

Reply:
done